pkg/operators/kubemanager/kubemanager.go

```diff
@@ -117,10 +117,11 @@ func (k *KubeManager) ParamDescs() params.ParamDescs {
 			DefaultValue: "false",
 		},
 		{
-			Key:         ParamNamespace,
-			Alias:       "n",
-			Description: "Show only data from pods in a given namespace",
-			ValueHint:   gadgets.K8SNamespace,
+			Key:          ParamNamespace,
+			Alias:        "n",
+			Description:  "Show only data from pods in a given namespace",
+			ValueHint:    gadgets.K8SNamespace,
+			DefaultValue: "default",
```

Comment:
[This](https://github.com/inspektor-gadget/inspektor-gadget/blob/fd1702ec3451c044c10ade79eba7916ec3e9b451/cmd/kubectl-gadget/utils/flags.go#L77) seems to be the legacy way of getting the namespace - I don't know if it can just be replaced by that, but I think it's better to get it from kubeconfig than just using "default". WDYT?

Reply:
Its a good idea. But I don't have an idea on top of my head how we could achieve that in a general way.

Like having operator parameters with default values dependant on the client sides files. Did I miss something?

Reply:
Good point - I think the initial idea was to evaluate the `ValueHint` on the client side and set/replace defaults based on that.

Reply:
I removed that commit. Doesn't have to do anything with the issue this PR wants to solve

Reply:
> Good point - I think the initial idea was to evaluate the `ValueHint` on the client side and set/replace defaults based on that.

I think we're already doing it on https://github.com/inspektor-gadget/inspektor-gadget/blob/9e8b8dc8893bb878eed602568581ef59603a8ed4/cmd/kubectl-gadget/main.go#L102-L103